src/components/Confirm.js

```diff
@@ -66,13 +103,6 @@ function ConfirmationCheckbox() {
     setPerson(wives[randomNum]);
   };
 
-  const playAgain = () => {
```

Comment:
why are you removing the playAgain button? please don't remove it.

Reply:
Apology for removing it ,will be adding it back with all functionality for sure